plugins/org.eclipse.emfcloud.modelserver.glsp.integration/src/org/eclipse/emfcloud/modelserver/glsp/EMSGLSPServer.java

```diff
@@ -33,6 +33,7 @@ public class EMSGLSPServer extends DefaultGLSPServer {
    private static Logger LOGGER = LogManager.getLogger(EMSGLSPServer.class);
    private static final String TIMESTAMP_KEY = "timestamp";
    private static final String MODELSERVER_URL_KEY = "modelServerURL";
+   private static final int NUMBER_OF_TRIES = 10;
```

Comment:
Perhaps we could make this protected in case someone would like to extend the number of tries?

Reply:
its final, you can't override it. we could move this to a method.

Reply:
Ah of course, overlooked that, then nevermind! :)